internal/plugins/envtest/v1/init.go

```diff
@@ -47,19 +44,9 @@ func (s *initSubcommand) InjectConfig(c config.Config) error {
 }
 
 func (s *initSubcommand) Scaffold(fs machinery.Filesystem) error {
-	makefileBytes, err := afero.ReadFile(fs.FS, filePath)
+	err := sdkutil.ReplaceInFile(filePath, oldMakefileTestTarget, fmt.Sprintf(makefileTestTarget, controllerRuntimeVersion))
 	if err != nil {
-		return err
-	}
-
-	makefileBytes = []byte(strings.Replace(string(makefileBytes), oldMakefileTestTarget, fmt.Sprintf(makefileTestTarget, controllerRuntimeVersion), 1))
-
-	var mode os.FileMode = 0644
-	if info, err := fs.FS.Stat(filePath); err == nil {
```

Comment:
@Adirio , regarding the code implementation, suggested which I am ok within a follow-up. See that it was NOT raising the issue when the target is not found as it is in the Insert/replace functions that we used. Just to let, you know.

Reply:
It needs to be done in another pr.